src/views/PositionPlot/PositionPlotViewData.ts

```diff
@@ -0,0 +1,18 @@
+import { validateObject } from "figurl"
+import { isArrayOf, isEqualTo, isString } from "figurl/viewInterface/validateObject"
+
+export type PositionPlotViewData = {
+    type: 'PositionPlot'
+    timestamps: number[]
+    positions: number[][]
```

Comment:
My instinct is that some of the filtering code in PositionPlotView will be a bit more transparent if we make these into objects and index the objects, rather than having to keep the indices of several different lists synced up. (Not sure whether there'd be a noticeable performance impact to this.)

For the individual positions, is it safe to assume `positions[timestamp] => some_position[]` where  `some_position[0]` is the x-coordinate and `some_position[1]` is the y-coordinate? (And we probably don't anticipate a z-coordinate, but we could add it if we needed to?) It might be nice to map explicitly to named coordinate variables, but we would just have to map back to point vectors when we do the coordinate transform to get pixels, so I'm less fussed about that.